app/ui/src/main/res/layout/activity_account_settings.xml

```diff
@@ -4,7 +4,17 @@
               android:layout_height="fill_parent"
               android:layout_width="fill_parent">
 
-    <include layout="@layout/toolbar" />
+    <android.support.v7.widget.Toolbar
+                android:id="@+id/toolbar"
+                android:layout_width="match_parent"
+                android:layout_height="?attr/actionBarSize"
+                android:background="?attr/colorPrimary"
+                android:elevation="4dp">
+        <com.fsck.k9.ui.settings.account.AccountSelectionSpinner
+                    android:id="@+id/accountSpinner"
+                    android:layout_width="fill_parent"
```

Comment:
I've changed it to `wrap_content` as you requested
The original reasoning was to have the drop-down width be fixed to toolbar's width rather than the content. Personal preference, I suppose